static/app/views/settings/projectSourceMaps/projectSourceMaps.spec.tsx

```diff
@@ -342,13 +349,6 @@ describe('ProjectSourceMaps', function () {
           })
         );
       });
-
-      // Switch tab
-      await userEvent.click(screen.getByRole('link', {name: /Release Bundles/}));
-      expect(router.push).toHaveBeenCalledWith({
```

Comment:
We need to find out why this stopped working, but I'm not too concerned about it because we plan to remove these tabs eventually.